models/Asset/Image/Thumbnail/Config/Dao.php

```diff
@@ -106,7 +112,16 @@ protected function autoClearTempFiles()
     {
         $enabled = \Pimcore::getContainer()->getParameter('pimcore.config')['assets']['image']['thumbnails']['auto_clear_temp_files'];
         if ($enabled) {
-            $this->model->clearTempFiles();
+            $arguments = [
+                Console::getPhpCli(),
+                PIMCORE_PROJECT_ROOT.'/bin/console',
+                'pimcore:thumbnails:clear',
+                '--type=image',
+                '--name='.$this->model->getName()
+            ];
+
+            $process = new Process($arguments);
+            $process->start();
```

Comment:
This unfortunately doesn't work,  as the process doesn't get detached,  it is killed as soon as the main process ends, see also  #8371